src/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -212,7 +212,7 @@ export const VisibilityCard: React.FC<VisibilityCardProps> = props => {
                     },
                   })
                 }
-                error={getFieldError(errors, "isPublishedAt")}
+                error={getFieldError(errors, "isPublishedAt") as any}
```

Comment:
Suggestion: I would add a `TODO` or try to type it but it's not necessary 😸

Reply:
+1 IMHO using `@ts-expect-error` is better because it forces you to remove the label once the error is fixed ;)